container-runtime/gvisor/gvisor-kvm.part

```diff
@@ -0,0 +1,6 @@
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runsc-kvm]
+  runtime_type = "io.containerd.runsc-kvm.v1"
+
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runsc-kvm.options]
+  TypeUrl = "io.containerd.runsc-kvm.v1.options"
+  ConfigPath = "/etc/cri/conf.d/runsc-kvm.toml"
```

Comment:
```suggestion
  BinaryName = "/usr/local/bin/runsc"
  ConfigPath = "/etc/cri/conf.d/runsc-kvm.toml"
```

I think this should fix it, not entirely sure about the path to the runsc binary.

Reply:
nope, still same error

Reply:
I'm sorry, let me not waste your time and find a way to debug that myself. I'll come back once I got it running :)

Reply:
Found it, the `runtime_type` was the mistake, this doesn't need to be adjusted. Sorry for the chaos.